modules/monitoring-setting-up-pod-topology-spread-constraints-for-alertmanager.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-up-pod-topology-spread-constraints-for-alertmanager_{context}"]
+= Setting up pod topology spread constraints for Alertmanager
+
+For core {product-title} platform monitoring, you can set up pod topology spread constraints for Alertmanager to fine tune how pod replicas are scheduled to nodes across zones. 
+Doing so helps ensure that Alertmanager pods are highly available and run more efficiently because workloads are spread across nodes in different data centers or hierarchical infrastructure zones.
+
+You configure pod topology spread constraints for core platform monitoring in the `cluster-monitoring-config` config map.
```

Comment:
You configure pod topology
->
You can configure pod topology

Reply:
fixed